### PR TITLE
Fb api query params

### PR DESCRIPTION
**Pull request by adantoscano**
I've added query object to facebook api class to allow request with query param. I've created two new methods to make it more readable, `bot.api.callAPI()` is redundant, `bot.api.get(path, query)` and `bot.api.post(path, body, query)` use callAPI in a convenient way.

This is a way to fix my issue #1767

### CODE REVIEW

**Comment by benbrown** [comment]
@adantoscano is it ever necessary to use the payload AND the query params? 

Instead of adding a new parameter and 2 new methods, what about just using the payload to hold query params when it is a GET?

**Comment by adantoscano** [comment]
Done @benbrown

File: packages/botbuilder-adapter-facebook/src/facebook_api.ts [hunk 1]
```diff
@@ -47,15 +47,26 @@ export class FacebookAPI {
...
      */
-    public async callAPI(path: string, method: string = 'POST', payload: any): Promise<any> {
-        let proof = this.getAppSecretProof(this.token, this.secret);
+    public async callAPI(path: string, method: string = 'POST', payload: any = {}): Promise<any> {
+        const proof = this.getAppSecretProof(this.token, this.secret);
+
+        let queryString = '?';
+        let body = {};
+
+        if (method === 'GET') {
```
Comment (benbrown) [inline packages/botbuilder-adapter-facebook/src/facebook_api.ts]: use toUpper here so the method is not case sensitive

File: packages/botbuilder-adapter-facebook/src/facebook_api.ts [hunk 2]
```diff
@@ -47,15 +47,26 @@ export class FacebookAPI {
...
+            for(const key in payload) {
+                queryString = queryString + `${key}=${payload[key]}&`;
```
Comment (benbrown) [inline packages/botbuilder-adapter-facebook/src/facebook_api.ts]: should use encodeURIComponent for these

**Comment by adantoscano** [comment]
Done!

**Pull request merged**